.eslintrc

```diff
@@ -0,0 +1,137 @@
+{
+  "parser": "babel-eslint",
+
+  "ecmaFeatures": {
+    "modules": true,
+    "experimentalObjectRestSpread": true
+  },
+
+  "env": {
+    "browser": false,
+    "es6": true,
+  },
+
+  "rules": {
+    // specify the maximum depth that blocks can be nested
+    "max-depth": [0, 4],
+    // specify the maximum length of a line in your program
+    "max-len": [0, 80, 4],
```

Comment:
could you clarify how we can read the config `[0, 80, 4]` - min 0 characters and 80 max? and what for `4` is here? confused a bit , thanks.